internal/tests/api/wallet_test.go

```diff
@@ -253,8 +255,37 @@ func TestWalletSend_EthSuccess(t *testing.T) {
 
 	require.True(t, response.Data.CanSend)
 	require.False(t, response.Data.InsufficientBalance)
-	t.Logf("Successfully sent ETH")
+	t.Logf("Successfully generated transaction info for sending ETH")
+
+	sub, _ := eth.CreateTxSubmissionFromInfo(response.Data.TxInfo, nil)
+	submitResponse, err := apiClient.Tx.SubmitTx(sub, nil, eth.GweiToWei(10), eth.GweiToWei(1))
+	require.NoError(t, err)
+	t.Log("SubmitTx called")
+
+	err = testMgr.CommitBlock()
+	require.NoError(t, err)
+
+	_, err = apiClient.Tx.WaitForTransaction(submitResponse.Data.TxHash)
+	require.NoError(t, err)
+	t.Log("Waiting complete")
+
+	// Check the balance
+	tm, err := osha.NewTestManager()
```

Comment:
There's a global variable in the package called `testMgr` - use that instead of making your own test manager.

Reply:
no longer need tm since sp.GetEthClient() handles this